### ISSUE TITLE
Packer: Break out linting

### ISSUE THREAD

**Issue by mamercad** [StackStorm/packer-st2#66]
Naturally it's always nicer to have the linting checks as a dedicated jobs in the CI pipeline like it was before (`inspec-lint`, `packer-lint` running in parallel as a dependency to the main "packer-build" task). Provides some better visuals and less monolytics.

But because there was already a lot of work, not a requirement. Just nice to have, if you'll have a few more cycles to spend on it.

_Originally posted by @armab in https://github.com/StackStorm/packer-st2/pull/64#discussion_r1086606322_
